docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -362,7 +373,9 @@ processor.
       .. code-block:: console
 
          user@client$ sawtooth settings list
-         sawtooth.settings.vote.authorized_keys: 0276023d4f7323103db8d8683a4b7bc1eae1f66fbbf79c20a51185f589e2d304ce
+         sawtooth.consensus.algorithm.name: Devmode
+         sawtooth.consensus.algorithm.version: 0.1
+         sawtooth.settings.vote.authorized_keys: 0276023d4f7323103db8d8683a4b7bc1eae1f66...
```

Comment:
Note that I changed the example output to be what the actual output now looks like (with a truncated key ending with `..`)